### PR TITLE
Bhosle Mahesh 69,83

### PR DESCRIPTION
**Pull request by Bhosle07**

### CODE REVIEW

File: SubmittedAssignments/Bhosle_Mahesh_83/css/style.css [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    margin-top: -13mm;
+    margin-left: -33mm;
+    overflow: hidden;
+    position: absolute;
+}
+
+.tri1{
+    position: absolute;
+    width: 313px;
+    height: 2px;
```
Comment (KalpeshApnasite) [inline SubmittedAssignments/Bhosle_Mahesh_83/css/style.css:50]: don't use px

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]
don't use px

File: SubmittedAssignments/Bhosle_Mahesh_83/css/style.css [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    left: 40mm;
+    bottom: 9mm;
+    position: absolute;
+    font-size: 65pt;
+    color: rgb(254, 202, 82);
+    border-bottom: 2px solid rgb(254, 202, 82);
+}
+.main-content{
+    font-size: 13.5pt;
+    letter-spacing: 1px;
```
Comment (KalpeshApnasite) [inline SubmittedAssignments/Bhosle_Mahesh_83/css/style.css:167]: do not use px

File: SubmittedAssignments/Bhosle_Mahesh_69/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<head>
+	<title>
+		Certificate
+	</title>
+	<link rel="stylesheet" type="text/css" href="./css/style.css">
+	<link rel="stylesheet" type="text/css" href="./css/bootstrap.min.css">
+
+</head>
+
+<body>
```
Comment (KalpeshApnasite) [inline SubmittedAssignments/Bhosle_Mahesh_69/index.html:13]: add only certificate-background in index.html

File: SubmittedAssignments/Bhosle_Mahesh_83/.vscode/settings.json
Comment (KalpeshApnasite) [inline SubmittedAssignments/Bhosle_Mahesh_83/.vscode/settings.json:1]: remove this file

File: SubmittedAssignments/Bhosle_Mahesh_crt_76.jpg/css/style.css
Comment (KalpeshApnasite) [inline SubmittedAssignments/Bhosle_Mahesh_crt_76.jpg/css/style.css:1]: follow folder name format

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]

**Review by Adarshovhal — COMMENTED** [review]
In css file mm and pt unit are used. Not px & em used anywhere.
All classes in the previous.html file.
Background classes in the index.html file.
Code was good in structure.

**Pull request merged**